pkgs/applications/networking/eddie-ui/default.nix

```
@@ -0,0 +1,92 @@
+{ stdenv,
+  fetchFromGitHub,
+  clang,
+  cmake,
+  mono,
+  dotnet-sdk,
+  which,
+  glibc,
+  pkgconfig,
+  gtk2,
+  libappindicator-gtk2,
+  lzma,
+  libselinux,
+  libsepol,
+  fribidi,
+  libthai,
+  libdatrie,
+  xorg,
+  openvpn,
+  stunnel
+  }:
+
+stdenv.mkDerivation rec {
+  pname = "eddie-ui";
+  version = "2.18.9";
+  arch = if stdenv.system == "i686-linux"
+    then "x86"
+    else "x64";
+  src = fetchFromGitHub {
+    owner = "AirVPN";
+    repo = "Eddie";
+    rev = "2.18.9";
```

```suggestion
    rev = version;
```

Otherwise we won't get auto updates.